recipe/meta.yaml

```diff
@@ -1,53 +1,52 @@
-{% set version = "2.0.2" %}
+{% set version = "0.8.4" %}
 
 package:
   name: mistune
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/m/mistune/mistune-{{ version }}.tar.gz
-  sha256: 6fc88c3cb49dba8b16687b41725e661cf85784c12e8974a29b9d336dd596c3a1
+  sha256: 59a3429db53c50b5c6bcc8a07f8848cb00d7dc8bdb431a4ab41920d201d4756e
 
 build:
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 1000
+  # trigger 1
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
+  build:
+    - {{ compiler('c') }}
   host:
+    - cython
     - python
     - pip
-    - setuptools
-    - wheel
   run:
     - python
-  run_constrained:
-    # 6.1.0 pins to mistune <2 (a future version may loosen this pin)
-    # see: https://github.com/conda-forge/nbconvert-feedstock/pull/54
-    - nbconvert >6.1.0
```

Comment:
Can we really remove this? Are we going to remove the 2.0.2 packages from the repo?

Reply:
Probably we can remove it. We aren't going to remove the 2.0.2 packages from the repo.

Reply:
Hah, I misunderstood how this works! Sorry :)